### ISSUE TITLE
Discs connected to LSI Logic MegaRAID SAS RAID Module don’t get recognized.

### ISSUE THREAD

**Issue by MrSchnirschuh** [IceWhaleTech/ZimaOS#130]
I use in My R720xd and 2x R720 H710P Mini in IT mode and H310P Mini in IT mode and Zima OS doesn’t recognize my SAS HDDs.

**Comment by ArkPlays** [comment]
I am running into the same problem with a Dell R440.

**Comment by Wanneshey** [comment]
Did you end up finding a solution for this?

**Comment by MrSchnirschuh** [comment]
> Did you end up finding a solution for this?

Hi, I currently don't have Zima OS installed anymore, so I haven't tried to find a solution since then. Are you using the latest version of Zima OS?

**Comment by Wanneshey** [comment]
Yes. ZimaOS has some missing drivers so I have to find workarounds or ask to get it patched

**Comment by MrSchnirschuh** [comment]
If I remember correctly, an alternative like the Fujitsu 9300-8I IT Mode FW: LSI SAS3008 12GB HBA Card works. Have you tried a different HBA like that one? 

Also, are the missing drivers specifically for SAS or SATA? If you only have 2 drives, you could just use the internal SATA ports on the motherboard as a workaround. I think that depends on whether you are using the R720 or the R720xd, though.
